addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -358,6 +356,19 @@ var FormRenderer = BasicRenderer.extend(WidgetAdapterMixin, {
             });
         }
     },
+    _applyFormSizeClass: function () {
+        const formEl = this.$el[0];
+        if (config.device.size_class <= config.device.SIZES.XS) {
+            formEl.classList.add('o_xxs_form_view');
+        } else {
+            formEl.classList.remove('o_xxs_form_view');
+        }
+        if (config.device.size_class >= config.device.SIZES.XXL) {
+            formEl.classList.add('o_xxl_form_view');
+        } else {
+            formEl.classList.remove('o_xxl_form_view');
+        }
```

Comment:
Just wondering why not just string equality on them? Future-proof on having Odoo on a single pixel/unlimited pixels? /s

Reply:
lol :D
I don't know, I took inspiration from existing code, I'm not really sure how we want these sizes to work....